docs/index.asciidoc

```diff
@@ -181,7 +181,8 @@ The fingerprint method to use.
 
 If set to `SHA1`, `SHA256`, `SHA384`, `SHA512`, or `MD5` and a key is set,
 the cryptographic hash function with the same name will be used to generate
-the fingerprint. When a key set, the keyed-hash (HMAC) digest function will
+the fingerprint. 
+If no key is set, the keyed-hash (HMAC) digest function will
```

Comment:
"When a key set, the keyed-hash (HMAC) digest function will the fingerprint" is right
If no key is set, HMAC will not be used. I am confused

maybe rephase as following?
"If set to `SHA1`, `SHA256`, `SHA384`, `SHA512`, or `MD5` and a key is set,
the cryptographic hash function with the same name will be used. 
Then, the keyed-hash (HMAC) digest function will generate the fingerprint"